ocs_ci/ocs/node.py

```diff
@@ -395,21 +395,30 @@ def get_node_resource_utilization_from_adm_top(nodename=None, node_type='worker'
     for node in node_names:
         for value in resource_utilization_all_nodes:
             if node in value:
-                value = re.findall(r'\d+', value.strip())
-                cpu_utilization = value[2]
+                value = re.findall(r'(\d{1,3})%', value.strip())
+                cpu_utilization = value[0]
                 log.info("The CPU utilized by the node "
                          f"{node} is {cpu_utilization}%")
-                memory_utilization = value[4]
+                memory_utilization = value[1]
                 log.info("The memory utilized of the node "
                          f"{node} is {memory_utilization}%")
                 utilization_dict[node] = {
                     'cpu': int(cpu_utilization),
                     'memory': int(memory_utilization)
                 }
+
+    if print_table:
+        usage_memory_table = PrettyTable()
+        usage_memory_table.field_names = ["Node Name", "CPU USAGE adm_top",
+                                          "Memory USAGE adm_top"]
+        for node, util_node in utilization_dict.items():
+            usage_memory_table.add_row([node, f'{util_node["cpu"]}%', f'{util_node["memory"]}%'])
+        log.info(f'\n{usage_memory_table}\n')
     return utilization_dict
 
 
-def get_node_resource_utilization_from_oc_describe(nodename=None, node_type='worker'):
+def get_node_resource_utilization_from_oc_describe(nodename=None,
```

Comment:
break the function according to coding guidelines
https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md